specification/schemas/PatchShipmentSurcharge.json

```diff
@@ -0,0 +1,41 @@
+{
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/MinimalShipmentSurcharge"
+    },
+    {
+      "required": [
+        "attributes"
+      ],
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "required": [
+            "amount",
+            "currency"
```

Comment:
Why are both amount and currency required when making a change here? Shouldn't you be able to just send the amount if the currency stays the same?